test/utils/MockAuthenticatedAction.scala

```diff
@@ -29,6 +30,9 @@ import scala.concurrent.Future
 class MockAuthenticatedAction @Inject()(override val authConnector: AuthConnector) extends AuthenticatedActionRefiner(authConnector) {
 
   override protected def refine[A](request: Request[A]): Future[Either[Result, AuthenticatedUserRequest[A]]] = {
+
+    val nino = request.body
```

Comment:
whats this for?

Reply:
Test code left in place. Good spot I have removed it.